lib/environment.rb

```diff
@@ -267,8 +267,8 @@ def system_docker_repo
     # helper parser
     def parse_version(ver_str)
       ver = ver_str.sub(/(^v|^openshift-clients-)/,"")
-      if ver !~ /^[\d.]+$/
-        raise "version '#{ver}' does not match /^[\d.]+$/"
+      if ver !~ /^[\d][.][\d].*$/
+        raise "version '#{ver}' does not match /^[\d][.][\d].*$/"
       end
       ver = ver.split(".").reject(&:empty?).map(&:to_i)
```

Comment:
maybe we need to change this as well if we want to support version of the format `4.5-whatever`. If we are sure that we get a dot or end of string after the second number, then doesn't matter.

Reply:
actually ignore, it doesn't matter. 
```
[7] pry(main)> "32-d243".to_i
=> 32
```